common.mk

```diff
@@ -5,7 +5,7 @@ VERSION=$(shell git describe --tags --always)
 init: init-api-tools
 	go install github.com/google/wire/cmd/wire@latest
 	go install github.com/vektra/mockery/v2@v2.20.0
-	go install ariga.io/atlas/cmd/atlas@v0.12.0
+	curl -sSf https://atlasgo.sh | sh -s -- -y
```

Comment:
It would be great if you could add a comment explaining why we need to download the binary instead of using `go install`. I link to the comment in the issue where they say that ent is not part of the code https://github.com/ariga/atlas/issues/2388#issuecomment-1864287189 would suffice

Reply:
Sure. Explanation added, thanks.